sklearn/decomposition/tests/test_nmf.py

```diff
@@ -86,8 +86,8 @@ def test_initialize_variants():
 @ignore_warnings(category=UserWarning)
 def test_nmf_fit_nn_output():
     # Test that the decomposition does not contain negative values
-    A = np.c_[5 * np.ones(5) - np.arange(1, 6),
-              5 * np.ones(5) + np.arange(1, 6)]
+    A = np.c_[np.full(5, 5.) - np.arange(1, 6),
+              np.full(5, 5.) + np.arange(1, 6)]
```

Comment:
Actually, here "np.full(5, 5.)" can be replaced by "5." thanks to broadcasting.

Reply:
Good point!